dts/bindings/spi/nordic,nrf-spis.yaml

```diff
@@ -5,7 +5,7 @@ description: Nordic nRF family SPIS (SPI slave with EasyDMA)
 
 compatible: "nordic,nrf-spis"
 
-include: nordic,nrf-spi-common.yaml
+include: ["nordic,nrf-spi-common.yaml", "memory-region.yaml"]
```

Comment:
Why is this actually added in this PR? It does not seem to be related.

Reply:
Moved to another PR.